hil/ext/switches/n3000.py

```diff
@@ -39,13 +44,13 @@ class DellN3000(Switch):
 
     @staticmethod
     def validate(kwargs):
-        schema.Schema({
+        Schema({
             'username': basestring,
             'hostname': basestring,
             'password': basestring,
-            'dummy_vlan': schema.And(schema.Use(int),
-                                     lambda v: 0 < v and v <= 4093,
-                                     schema.Use(str)),
+            'dummy_vlan': And(Use(int),
+                              lambda v: 0 < v and v <= 4093,
```

Comment:
While you're at it, this python supports a shorthand, so this can be `0 < v <= 4093`.

Also, is there a reason for 4093? The others are 4096, which makes more sense.

Reply:
yeah, the Dell N3000 switch only showed that it's from 1 to 4093. 
```
R5-PA-C01-U39#show vlan id ?

<1-4093>                 Enter VLAN ID.

R5-PA-C01-U39#show vlan id 4094
                           ^
Value is out of range. The valid range is 1 to 4093.
```

Reply:
Weird. Would you mind adding a comment to that effect?

Reply:
In [network-drivers](https://github.com/CCI-MOC/hil/blob/master/docs/network-drivers.md#switch-preparation-1) it's mentioned that we enable VLAN 2-4093, but it doesn't mention the reason.  @Izhmash, can you put a comment in the code while you are at it and update the docs to make it more clear?